packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/sensororientation/DeviceOrientationManager.java

```diff
@@ -159,6 +159,9 @@ public int getVideoOrientation() {
    *
    * <p>Returns one of 0, 90, 180 or 270.
    *
+   * <p>More details can be found in the official Android documentation:
+   * https://developer.android.com/training/camera2/camera-preview-large-screens#orientation_calculation
```

Comment:
The linked documentation specifically discusses both clockwise and counterclockwise rotation, so it doesn't really clarify the thing I was asking about. This comment should say whether the rotation returned is clockwise or counterclockwise, and why.

In fact, based on that documentation it's not clear to me that this method is even intended to do. Is it supposed to apply to front-facing or back-facing cameras? Or is it both, in which case this PR is as wrong as the original code, just for the opposite cameras.

Reply:
According to the docs for `MediaRecorder.setOrientationHint(int)` the rotation is in clockwise degrees. That documentation does not specifically mention a front or back facing camera, so I assume it refers to both? That paragraph is rather slim when it comes to actual information, but this method is the only consumer of the computation, so I don't really see what else could be of use in terms of docs?

Reply:
> Or is it both, in which case this PR is as wrong as the original code, just for the opposite cameras.

I tested the fix on my phone with both front & back camera's in both landscape left and right. All situations did produce the correct results. I could verify the fix on a tablet as well, just to be sure?

Reply:
Resolving since the first of the two links explains that it's clockwise; I missed that before. I am still confused as to why front-facing isn't opposite, but maybe it's specifically to avoid this kind of confusion.